staging/cse/windows/provisioningscripts/kubeletstart.ps1

```diff
@@ -48,6 +55,29 @@ if ($global:KubeBinariesVersion -lt "1.27.0") {
     $KubeletArgList += @("--container-runtime=remote")
 }
 
+# Run the secure TLS bootstrap script to generate a kubelet client certificate, if enabled
+if ($global:EnableSecureTLSBootstrapping) {
+    $aadResource = $global:DefaultSecureTLSBootstrapAADResource
+    if ($global:CustomSecureTLSBootstrapAADResource) {
+        $aadResource = $global:CustomSecureTLSBootstrapAADResource
+    }
+    & "c:\k\securetlsbootstrap.ps1" -KubeDir $global:KubeDir -APIServerFQDN $global:MasterIP -AADResource $aadResource
+    if (!$?) {
+        Write-Host "Secure TLS bootstrapping failed, will still try to start kubelet..."
```

Comment:
what will happen if there is no valid cert/token?

Reply:
if there's no valid cert after secure bootstrapping then we'd fall-back to the token supplied by RP

if the token itself is invalid, then the node wouldn't be able to join the cluster... though that's what the case is currently since we always use a bootstrap token as far as I know

after the first 2 rollout phases are completed, we plan on removing/disabling bootstrap tokens _only_ once we see high qos/good enough signals from the secure protocol